sdk/identity/identity/src/credentials/deviceCodeCredential.ts

```diff
@@ -127,10 +127,12 @@ export class DeviceCodeCredential implements TokenCredential {
       scopes: scopeArray
     };
 
-    logger.info("Sending devicecode request");
+    logger.info(`DeviceCodeCredential invoked. Scopes: ${scopeArray.join(", ")}`);
 
     try {
-      return this.acquireTokenByDeviceCode(deviceCodeRequest);
+      const token = await this.acquireTokenByDeviceCode(deviceCodeRequest);
+      logger.info(`DeviceCodeCredential succeeded. Scopes: ${scopeArray.join(", ")}. Expires on timestamp: ${token?.expiresOnTimestamp}`);
```

Comment:
succeeded in retrieving the access token. Scopes ? ?

Reply:
I like the second one! Thank you

Reply:
Did the change! Please review again.